rep-guides/warm-outbound-minset.mdx

```diff
@@ -6,6 +6,6 @@ description: "Description here"
 <video
   controls
   className="w-full aspect-video"
-  src="https://share.descript.com/embed/BmcMWUvprX3"
+  src="https://vimeo.com/944510477"
```

Comment:
Does this need to be the `player` url domain for vimeo? I wonder if this will work without it.